### ISSUE TITLE
API endpoint mismatch between documentation and examples

### ISSUE THREAD

**Issue by alecalve** [questdb/questdb.io#286]
URL: https://questdb.io/docs/develop/insert-data#query-endpoint

Documentation uses `/query` path but code examples use `/exec`.

**Comment by mpsq** [comment]
@bsmth 
As Antoine correctly highlighted, `query` in that page should be `exec` (in the title above the code examples).

Not directly related to the typo that Antoine found, we could potentially improve this page by:

- Explaining that inserting data like that will create the table on the fly
- Showing how to create a table in case a user wants to manually specify types

A table can be created either via REST or Postgres wire but not InfluxDB line protocol.

What do you think?

**Comment by bsmth** [comment]
Thanks @mpsq , I started working on this already at https://github.com/questdb/questdb.io/pull/287

**Comment by alecalve** [comment]
@mpsq 

tables aren't created on the fly via the REST API.

```
$ curl -G --data-urlencode "query=INSERT INTO trades VALUES('abc', 123456);" http://localhost:9000/exec
{"query":"INSERT INTO trades VALUES('abc', 123456);","error":"table 'trades' does not exist","position":12}
```

**Comment by mpsq** [comment]
Sorry, @alecalve I edited my comment too late, this is only the case for the InfluxDB line protocol endpoint.

**Comment by bsmth** [comment]
Thanks for the issue, @alecalve - fixed in #287
